packages/react-components/src/AccessibilityScenarios/Input.stories.tsx

```diff
@@ -0,0 +1,209 @@
+import * as React from 'react';
+
+import { Input, InputOnChangeData } from '@fluentui/react-input';
+import { Label } from '@fluentui/react-label';
+import { Button } from '@fluentui/react-button';
+
+import { Scenario } from './utils';
+
+export const RegistrationFormInputsAccessibilityScenario = () => {
+  // eslint-disable-next-line
+  const validations: Record<string, any> = {
+    fullName: {
+      type: 'field',
+      regexes: [/^[A-Za-zÀ-ÖØ-öø-ÿěščřžďťňůĚŠČŘŽĎŤŇŮ -]{2,50}$/],
+    },
+    nickname: {
+      type: 'field',
+      regexes: [/^[A-Za-zÀ-ÖØ-öø-ÿěščřžďťňůĚŠČŘŽĎŤŇŮ -]{2,20}$|^$/],
+    },
+    password: {
+      type: 'password',
+      regexes: [
+        /^[^ ]{8,30}$/,
+        /^[^ ]*[0-9][^ ]*$/,
+        /^[^ ]*[A-Z][^ ]*$/,
+        /^[^ ]*[a-z][^ ]*$/,
+        /^[^ ]*[^A-Za-zÀ-ÖØ-öø-ÿěščřžďťňůĚŠČŘŽĎŤŇŮ ][^ ]*$/,
+      ],
+    },
+  };
+
+  const [values, setValues] = React.useState<Record<string, string>>({
```

Comment:
With this many interconnected state values, I wonder if `useReducer` would be more appropriate?